website/src/types/index.ts

```diff
@@ -105,4 +105,11 @@ export interface CompareResult {
 export interface CompareData {
   type: string;
   result: CompareResult;
-}
\ No newline at end of file
+}
+
+export type DailySummarydata = {
+  name: string;
+  data : { total_qps: MacroDataValue }[];
+}
+
+export type Workloads = 'OLTP' | 'OLTP-READONLY' | 'OLTP-SET' | 'TPCC' | 'TPCC_FK' | 'TPCC_UNSHARDED' | 'TPCC_FK_UNMANAGED'
```

Comment:
Unfortunately the list of workloads should be dynamic, they can change from time to time and is managed by the API itself. I do not think there is a route that just list the supported workload, but we can easily add that if you want.

Reply:
I understand what you are trying to do by typing this, but I am unsure if it's a sustainable option.